### ISSUE TITLE
nuxt-i18n support (lazy)

### ISSUE THREAD

**Issue by thefoxie** [nuxt/vite#58]
### Versions

nuxt-vite: 0.0.20
nuxt: 2.15.2

### Reproduction

I tried to make it in codesandbox, but it might work differently because there's no issue.
### Description

```
Uncaught SyntaxError: import not found: klona            plugin.main.js:39:9
```
Something like: https://github.com/nuxt-community/i18n-module/issues/1004

Tried to remove node_modules and .nuxt, add klona into my dependencies and peerDependencies, even tried the alias. Nothing worked.

It's being imported like
```
import { klona } from '/@fs/home/foxie/Code/project/node_modules/klona/full/index.js'
```
Why is there the /@fs prepended?

**Comment by pi0** [comment]
Hi @thefoxie. It seems alias is working but something wrong with vite. Can you please try [this workaround](https://vite.nuxtjs.org/misc/common-issues#uncaught-syntaxerror-the-requested-module--does-not-provide-an-export-named-default) for klona?

**Comment by thefoxie** [comment]
@pi0 tried that already and it doesn't make any difference :/

**Comment by pi0** [comment]
You are right. Issue was using commonjs build of klona. I've made PR to fix in i18n-module and a hotfix that makes it working v0.0.21 out of the box

**Comment by thefoxie** [comment]
@pi0 there's one more thing I suppose. It's linked to lazy option.
```
ReferenceError: require is not defined
```
```js
const { loadLanguageAsync } = require('./utils')
```

**Comment by pi0** [comment]
#1080

**Comment by pi0** [comment]
Should be fixed in nuxt-i18n@6.20.4 (https://github.com/nuxt-community/i18n-module/commit/f03f3433650762784937e50092437fc371b9bc3a)
